kernel/src/table_changes/physical_to_logical.rs

```diff
@@ -104,15 +104,21 @@ pub(crate) fn get_cdf_transform_expr(
     let cdf_values = get_cdf_columns(&state_info.logical_schema, scan_file)?;
     partition_values.extend(cdf_values);
 
-    get_transform_expr(transform_spec, partition_values, physical_schema)
+    get_transform_expr(
+        transform_spec,
+        partition_values,
+        physical_schema,
+        None, /* base_row_id */
```

Comment:
Can you add a TODO issue to get the base_row_id for CDF?

Reply:
@DrakeLin  this also indicates that we may want a `is_cdf_supported(TableFeature::RowTracking)`

Reply:
ah you're right:
https://docs.databricks.com/aws/en/delta/row-tracking#limitations

I think this is a bit odd tho. It's like not being able to access your primary key during a CDC. In any case, let's block now. Good callout 👍

Reply:
Can you add a note to `get_cdf_transform_expr` that makes it clear that we do not support reading row tracking info during CDF and that this is a known delta limitation? I want to avoid losing this context :)